examples/user_data/templates/windows_custom.tpl

```diff
@@ -0,0 +1,10 @@
+# Custom user data template provided for rendering
+<powershell>
+${pre_bootstrap_user_data ~}
+[string]$EKSBinDir = "$env:ProgramFiles\Amazon\EKS"
+[string]$EKSBootstrapScriptName = 'Start-EKSBootstrap.ps1'
+[string]$EKSBootstrapScriptFile = "$EKSBinDir\$EKSBootstrapScriptName"
+& $EKSBootstrapScriptFile -EKSClusterName ${cluster_name} ${bootstrap_extra_args} 3>&1 4>&1 5>&1 6>&1
```

Comment:
can we unify this with linux parameters to also use B64_CLUSTER_CA and API_SERVER_URL as then we wil be able to bootstrap node without asking AWS API (bootstrap scripts in case when parameters are not passed are doing awscli to gather required info)

Example:
```
& $EKSBootstrapScriptFile -EKSClusterName "windohs" -APIServerEndpoint "https://test.com" -Base64ClusterCA "dGVzdA==" -KubeletExtraArgs "--node-labels= --register-with-taints=" 3>&1 4>&1 5>&1 6>&1
```

Reply:
fixed in https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1680/commits/d380c9389f4ed61da1a51fa65099b9dd31a53438